momentum/4/config/ref-rollup-mx.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "rollup_mx"
+description: "If set to 'true' traffic for the associated host domain will be routed to the MX host of the destination host domain specified by the argument of the related rollup_mx_name option. This enhances the ability of Adaptive Delivery to respond to feedback from receiving domains. Usable in host stanzas, which can be included in global, binding or binding_group scopes."
```

Comment:
Same thing here, I think we want this in the description area below or some other section below, and this text should be a shortened version of that section. I see some of this text is in the Name section, but not all of it. I think we want less text here, and more of this information in the content below.

Reply:
The text here is from an earlier version of what is below.  Since it doesn't appear in the entry, and is (I think) just used for search, it is probably ok as is, but if you really want ot to be a duplicate of what is below, I can do that.